### PR TITLE
Update quickstart-for-actions-runner-controller.md

### LINKED ISSUE #34636: Installing Actions Runner Controller - Issue
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://github.com/github/docs/blob/main/content/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/quickstart-for-actions-runner-controller.md

### What part(s) of the article would you like to see updated?

Installing Actions Runner Controller
To install the operator and the custom resource definitions (CRDs) in your cluster, do the following.

In your Helm chart, update the NAMESPACE value to the location you want your operator pods to be created. This namespace must allow access to the Kubernetes API server.
Install the Helm chart.
The following example installs the latest version of the chart. To install a specific version, you can pass the --version argument along with the version of the chart you wish to install. You can find the list of releases in the [GitHub Container Registry](https://github.com/actions/actions-runner-controller/pkgs/container/actions-runner-controller-charts%2Fgha-runner-scale-set-controller).

NAMESPACE="arc-systems"
helm install arc \
    --namespace "{% raw %}${NAMESPACE}{% endraw %}" \
    --create-namespace \
    oci://ghcr.io/actions/actions-runner-controller-charts/gha-runner-scale-set-controller
For additional Helm configuration options, see [values.yaml](https://github.com/actions/actions-runner-controller/blob/master/charts/gha-runner-scale-set-controller/values.yaml) in the ARC documentation.

To enable ARC to authenticate to {% data variables.product.company_short %}, generate a {% data variables.product.pat_v1 %}. For more information, see [AUTOTITLE](https://github.com/github/docs/blob/main/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/authenticating-to-the-github-api#deploying-using-personal-access-token-classic-authentication).

### Additional information

The first part the process does not work.

If we just run the helm install, we get this error.
helm install arc --namespace arc-systems --create-namespace oci://ghcr.io/actions/actions-runner-controller-charts/gha-runner-scale-set-controller
Error: INSTALLATION FAILED: failed to authorize: failed to fetch anonymous token: unexpected status from GET request to https://ghcr.io/token?scope=repository%3Aactions%2Factions-runner-controller-charts%2Fgha-runner-scale-set-controller%3Apull&scope=repository%3Auser%2Fimage%3Apull&service=ghcr.io: 403 Forbidden

### PR DESCRIPTION
**Pull request by seilorjunior**
we need to be auth first, before the helm install (https://github.com/github/docs/issues/34636)

### Why:
To resolve this issue https://github.com/github/docs/issues/34636

Closes: 

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by Hazlizam850** [comment]
> we need to be auth first, before the helm install (#34636)
> 
> ### Why:
> To resolve this issue #34636
> 
> Closes:
> 
> ### What's being changed (if available, include any code snippets, screenshots, or gifs):
> ### Check off the following:
> * [x]  I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).
>   
>   * For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
> * [x]  For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

**Comment by nguyenalex836** [comment]
@seilorjunior Thanks so much for opening a PR! I'll get this triaged for review ✨

**Comment by jc-clark** [comment]
This looks great from a docs-content perspective. Once we have a confirmation from a subject matter expert, we can go ahead and merge this.

**Review by Ashrafn672 — COMMENTED** [review]
6 جي بي انټرنټ په  وړيا د 15 ورځو لپاره
فعالول 

1.  [@*666*452*1#``](g17d4d18604)

**Comment by Ku19** [comment]
_seilorjunior:patch-1_

**Review by tikto29 — COMMENTED** [review]
content/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/quickstart-for-actions-runner-controller.md

**Review by ghost — COMMENTED** [review]
Boo

File: content/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/quickstart-for-actions-runner-controller.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ In order to use ARC, ensure you have the following.
     The following example installs the latest version of the chart. To install a specific version, you can pass the `--version` argument along with the version of the chart you wish to install. You can find the list of releases in the [GitHub Container Registry](https://github.com/actions/actions-runner-controller/pkgs/container/actions-runner-controller-charts%2Fgha-runner-scale-set-controller).
 
     ```bash copy
+    gh auth token | helm registry login ghcr.io -u propyless --password-stdin
```
Comment (Link-) [inline content/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/quickstart-for-actions-runner-controller.md:50]: This is a public package and does not require authentication

**Comment by nguyenalex836** [comment]
@seilorjunior Thank you for your patience while our SME team reviewed! For the reasons listed above (and in the [linked issue](https://github.com/github/docs/issues/34636#issuecomment-2435492100)), we are unable to accept this contribution 💛 

Thank you for willingness to contribute to GitHub Docs! If you're interested, take a look at our [help wanted section](https://github.com/github/docs/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) to find open issues you can work on ✨

**Pull request closed without merging**